### ISSUE TITLE
npm outdated gives a wrong latest version number

### ISSUE THREAD

**Issue by patoi** [semantic-release/semantic-release#1400]
## Current behavior

`npm outdated` command's output is wrong: latest version is 15.4.0 but it is 15.14.0

<img width="356" alt="image" src="https://user-images.githubusercontent.com/1846548/71321085-0896c300-24b5-11ea-8bb7-9dcebe16eb9b.png">

I think the npmjs.com has a problem with version number: https://www.npmjs.com/package/semantic-release

The "Version" is 15.4.0 on npmjs.com, but "Last publish" value is "a day ago".

<img width="391" alt="image" src="https://user-images.githubusercontent.com/1846548/71321101-498ed780-24b5-11ea-8abb-7bdd18b9149f.png">

`npm ls semantic-release`: `semantic-release@15.14.0 `

`cat package.json| grep "semantic-release"`: `"semantic-release": "^15.14.0",`

`cat package-lock.json| grep "semantic-release"`: 

```
"semantic-release": {
      "resolved": "https://registry.npmjs.org/semantic-release/-/semantic-release-15.14.0.tgz",
```

## Expected behavior

Latest is 15.14.0

## Environment

- **semantic-release** version: 15.14.0 and 15.4.0

**Comment by gr2m** [comment]
`npm info semantic-release version` returns 15.4.0, it's the latest version. 15.14.0 is released using the `@next` dist-tag. So it all works as expected, try `npm info semantic-release@next version`. Let me know if you have any questions

**Comment by tuckbick** [comment]
This has not been the case though, right? This is new behavior. I have never used the `next` tag and have been pulling versions greater than `15.4.0` for months.

**Comment by tuckbick** [comment]
You can see similar behavior in the first two lines of the log in the issue [here](https://github.com/semantic-release/semantic-release/issues/1393) using `npx semantic-release`.

**Comment by patoi** [comment]
Ok, thank you!

I don't want to trolling, but IMHO it is self-contradictory. The project suggests the best practice:

https://github.com/semantic-release/semantic-release/blob/master/docs/support/FAQ.md#is-it-really-a-good-idea-to-release-on-every-push

**Comment by vxna** [comment]
@gr2m This really looks like new behavior.

I used it [through](https://github.com/vxna/optimize-three-webpack-plugin/blob/master/.travis.yml#L20) `npx semantic-release` two days ago and it was [version](https://travis-ci.com/vxna/optimize-three-webpack-plugin/jobs/269474289#L205) `15.13.31`. 
At the time of writing `npx semantic-release` return [version](https://travis-ci.com/vxna/optimize-three-webpack-plugin/jobs/269619419#L212) `15.4.0` which is more than [one year old](https://github.com/semantic-release/semantic-release/releases/tag/v15.4.0).

**Comment by gr2m** [comment]
@pvdlg do you know why `@next` is so far ahead of `@latest`?

**Comment by znerol** [comment]
I can confirm what @vxna observed. Some of my travis builds are breaking due to `npx` pulling in an ancient version of `semantic-release`. Example of a failing build:

* https://travis-ci.org/certhub/certhub-dehydrated-docker/jobs/628291717
      
      ...
      [Semantic release]: Running semantic-release version 15.4.0
      ...

Last successful build:

* https://travis-ci.org/certhub/certhub-dehydrated-docker/jobs/624353471
      
      ...
      [9:43:58 PM] [semantic-release] › ℹ  Running semantic-release version 15.13.31
      ...

The reason of the failure in my case is that this project does ship with a `package.json` (because it is not a javascript project). It ships with an appropriate `.releaserc.yml` which leaves out the `npm` plugin. However, that is only respected by more recent versions of `semantic-release`.

**Comment by pvdlg** [comment]
I added `15.4.0` to `@latest` by mistake yesterday without realizing it. It's corrected now and `15.14.0` is now the `@latest`.

**Comment by tuckbick** [comment]
Thanks for the quick fix @pvdlg!
